lib/index.ts

```diff
@@ -18,14 +18,20 @@ import {
 
   const deprecations = await crawlDeprecations(config);
 
-  const processors = [addGrouping, addUniqueKey];
-  const processedDeprecations = (await processors.reduce(
+  // Crawling Phase
+  const crawlingPhaseProcessors = [addUniqueKey, addCommentToRepository];
+  const processedCrawledDeprecations = (await crawlingPhaseProcessors.reduce(
     async (deps, processor) => await processor(config, await deps),
     Promise.resolve(deprecations)
   )) as Deprecation[];
+  await generateRawJson(config, processedCrawledDeprecations, { tagDate });
```

Comment:
This means that the group won't be included in the JSON output.
Is this OK?

Should we resave the JSON after the grouping phase?

Reply:
Done